fluentd/Dockerfile

```diff
@@ -33,7 +33,7 @@ RUN mkdir -p ${HOME} && \
       'elasticsearch-transport:<5' \
       'elasticsearch-api:<5' \
       'elasticsearch:<5' \
-      fluent-plugin-kubernetes_metadata_filter \
+      'fluent-plugin-kubernetes_metadata_filter:<1.0.0' \
```

Comment:
Is there a reason for this version restriction?

Reply:
yes, until we sort out why the new proposed cache code isn't working